klasses/models.py

```diff
@@ -0,0 +1,40 @@
+"""Models for klasses"""
+from django.db.models.base import Model
+from django.db.models.fields import (
+    DateTimeField,
+    DecimalField,
+    TextField,
+    IntegerField,
+)
+from django.db.models.fields.related import ForeignKey
+
+
+class Bootcamp(Model):
+    """
+    A bootcamp
+    """
+    title = TextField()
+
+
+class Klass(Model):
+    """
+    A class within a bootcamp
+    """
+    bootcamp = ForeignKey(Bootcamp)
+    klass_id = IntegerField()
+    price = DecimalField(max_digits=20, decimal_places=2)
+    start_date = DateTimeField(null=True)
+    end_date = DateTimeField(null=True)
```

Comment:
I think a title would help actually.  We'll display lists of them (maybe in redash) an title helps distinguish between them